kubemarine/procedures/backup.py

```diff
@@ -410,7 +410,13 @@ def _handle(self, payload: ParserPayload) -> None:
         items_by_resource: Dict[str, List[str]] = {}
 
         def append_item(api_version: str, kind: str, item: str) -> None:
-            resource_name = next(r['name'] for r in resources if r['apiVersion'] == api_version and r['kind'] == kind)
+            resource_name = next(
+                r['name'] for r in resources
+                if r['apiVersion'] == api_version
+                and (r['kind'] == kind
+                     or kind in ('ValidatingAdmissionPolicyList', 'ValidatingAdmissionPolicyBindingList')
+                     and r['kind'] == kind.rstrip('List'))
```

Comment:
Not an appropriate usage of `rstrip`, to be fixed
https://stackoverflow.com/questions/34544247/understanding-pythons-lstrip-method-on-strings